_includes/lesson.html

```diff
@@ -0,0 +1,27 @@
+<div class='lesson'>
+  <h2>#{{lesson.number}}
+    <small>
+      {{lesson.topic}}
+    </small>
+  </h2>
+
+  <p>{{lesson.description}}</p>
+
+  <ul>
+    {% for slide in lesson.slides %}
+      <li><a href="http://rubymonstas.ch/slides.rubymonstas.ch/slides/{{slide[0]}}/">Slides: {{ slide[1] }}</a> (<a href="https://github.com/rubymonstas-zurich/slides.rubymonstas.ch/releases/latest/download/{{slide[0]}}.pdf">PDF</a>)</li>
```

Comment:
Do I understand correctly that with the following yaml:

```yaml
slides:
     welcome: Welcome slides
```
`slide[0]` returns `welcome` and `slide[1]` `Welcome slides`?

Reply:
yes, exactly... liquid is a weird template language...